src/app/SBE/detail/cv/templates/header.js

```diff
@@ -17,11 +20,21 @@ const Header = ({ code, sbeCode, imageSrc, headerAttribute, actions }) => {
       </Box>
 
       <Spacer />
+
       <Box mt="6">
-        <Stack direction={['row', 'column']}>
-          {actions &&
-            map(action => <Action parentCode={sbeCode} code={action} targetCode={code} />)(actions)}
-        </Stack>
+        <Menu>
```

Comment:
why are we putting things behind context menus if we have the space?
maybe we should have a mobile switch that turns this in to context on mobiles?